android/build.gradle

```diff
@@ -76,6 +76,6 @@ dependencies {
     implementation 'com.facebook.react:react-native:+'
     implementation 'com.android.support:multidex:1.0.3'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation "com.google.android.libraries.navigation:navigation:5.3.1"
+    implementation "com.google.android.libraries.navigation:navigation:6.0.0"
```

Comment:
Nit: do we not define a specific KT/KGP version here? should we?

https://github.com/googlemaps/flutter-navigation-sdk/blob/8f0283ffc8885fa085b8c47edeeb2b6c44693e14/android/build.gradle#L27

Reply:
I updated the gradle version to 8.4.0 just like in the Flutter SDK.

As for Kotlin and Kotlin Gradle Plugin...I'm not sure if we need those definitions since our SDK is written in Java. That being said, adding those didn't cause any issues and maybe it's more future proof that way so I added them as well.